src/cache.js

```diff
@@ -46,6 +47,7 @@ export function updateAll(orgName) {
   }
   catch {
     // writeFileSync threw because the file doesn't exists.
+    logger.error("Failed to update orgs.json");
```

Comment:
```suggestion
  catch (error) {
    // writeFileSync threw because the file doesn't exists.
    logger.error(`Failed to update orgs.json: ${error}`);
```